docs/features.md

```diff
@@ -19,7 +19,11 @@ The [Cake.Issues.DupFinder addin] provides the following features:
 | <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IIssue.ProjectName`              |                                  |
```

Comment:
Even though not set in the issue provider there's now a `IIssue.Run` property. I added this also to the documentation with a remark that it can be set while reading issues:

```
| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IIssue.Run`                      | Can be set while reading issues  |
```

See e.g. [PR in Cake.Issues.InspectCode](https://github.com/cake-contrib/Cake.Issues.InspectCode/pull/127/files)